src/TestProblem/Hydro/CCSN/Init_TestProb_Hydro_CCSN.cpp

```diff
@@ -326,41 +332,42 @@ void SetParameter()
-      Aux_Message( stdout, "  central density threshold for CCSN_CC_MaxRefine_LV1 = %13.7e\n", CCSN_CC_MaxRefine_Dens1  ); }
+      Aux_Message( stdout, "  reduced maxmimum refinement lv 1                           = %d\n",     CCSN_CC_MaxRefine_LV1    );
+      Aux_Message( stdout, "  central density threshold for CCSN_CC_MaxRefine_LV1        = %13.7e\n", CCSN_CC_MaxRefine_Dens1  ); }
       if ( CCSN_CC_MaxRefine_Flag2 ) {
-      Aux_Message( stdout, "  reduced maxmimum refinement lv 2                    = %d\n",     CCSN_CC_MaxRefine_LV2    );
-      Aux_Message( stdout, "  central density threshold for CCSN_CC_MaxRefine_LV2 = %13.7e\n", CCSN_CC_MaxRefine_Dens2  ); }
-      Aux_Message( stdout, "  central density factor for reducing dt              = %13.7e\n", CCSN_CC_CentralDensFac   );
-      Aux_Message( stdout, "  reduced dt near bounce                              = %13.7e\n", CCSN_CC_Red_DT           ); }
-      Aux_Message( stdout, "  mode for rotational profile                         = %d\n",     CCSN_CC_Rot              );
+      Aux_Message( stdout, "  reduced maxmimum refinement lv 2                           = %d\n",     CCSN_CC_MaxRefine_LV2    );
+      Aux_Message( stdout, "  central density threshold for CCSN_CC_MaxRefine_LV2        = %13.7e\n", CCSN_CC_MaxRefine_Dens2  ); }
+      Aux_Message( stdout, "  central density factor for reducing dt                     = %13.7e\n", CCSN_CC_CentralDensFac   );
+      Aux_Message( stdout, "  reduced dt near bounce                                     = %13.7e\n", CCSN_CC_Red_DT           ); }
+      Aux_Message( stdout, "  mode for rotational profile                                = %d\n",     CCSN_CC_Rot              );
       if ( CCSN_CC_Rot == 1 ) {
-      Aux_Message( stdout, "  characteristic rotational radius R_0 (in cm)        = %13.7e\n", CCSN_CC_Rot_R0           );
-      Aux_Message( stdout, "  central angular frequency Omega_0 (in rad/s)        = %13.7e\n", CCSN_CC_Rot_Omega0       ); }
+      Aux_Message( stdout, "  characteristic rotational radius R_0 (in cm)               = %13.7e\n", CCSN_CC_Rot_R0           );
+      Aux_Message( stdout, "  central angular frequency Omega_0 (in rad/s)               = %13.7e\n", CCSN_CC_Rot_Omega0       ); }
       if ( CCSN_CC_Rot == 2 )
-      Aux_Message( stdout, "  multiplication factor for rotational profile        = %13.7e\n", CCSN_CC_Rot_Fac          );
-      Aux_Message( stdout, "  minimum angular resolution (in degree)              = %13.7e\n", CCSN_AngRes_Min/DEG2RAD  );
-      Aux_Message( stdout, "  maximum angular resolution (in degree)              = %13.7e\n", CCSN_AngRes_Max/DEG2RAD  );
+      Aux_Message( stdout, "  multiplication factor for rotational profile               = %13.7e\n", CCSN_CC_Rot_Fac          );
+      Aux_Message( stdout, "  radius within to refine to the allowed maximum level in km = %13.7e\n", CCSN_MaxRefine_Rad       );
```

Comment:
Change to "radius within which to refine to the maximum allowed level (in km)

Reply:
I have changed the comment.